.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: Test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows

https://github.com/orgs/community/discussions/24567